source/_docs/drupal-updates.md

```diff
@@ -27,20 +28,58 @@ If you are not having much luck with update.php, consider setting up the new sit
 
-### Content and Configuration
+1. Create a new Drupal 8 site on Pantheon from your User Dashboard.
+2. Add the 8.x version of your contrib modules. Some of these modules will have built-in migrating functionality that will help move their data from Drupal 7 to Drupal 8.
+3. Use the [Migrate](https://www.drupal.org/project/migrate) module to move over data and configuration from Drupal 6 or 7.
+4. Depending on the complexity of your site, you will likely want to review, revise, and rerun your migration.
+
+### Content and configuration
+
+Drupal 8 migrations automatically create the needed content types and establish the mappings between the old and new fields by default. You should review the configuration produced by these migrations by exporting your configuration to `yml` files ([a best practice for any Drupal 8 site](/docs/drupal-8-configuration-management/)).
+
+### Customizing migrations
+
+If needed, you can customize your migration using hooks and plugins. See the [drupal.org developer documentation](https://www.drupal.org/node/2127611) for more details.
+
+
+### Scripting migrations
 
-Drupal 8 migrations automatically create the needed content types and establish the mappings between the old and new fields by default.
+Depending on the complexity of your site, there is a good chance you will need to script and rerun migrations.
+We have [an example repository](https://github.com/stevector/migrate_pantheon) that shows how all the steps of a migration (from first configuring the migration to running it) can be done with Drush.
+
+The critical commands are:
+
+
+```
+terminus drush my-drupal-8-site.dev -- migrate-upgrade --legacy-db-key=drupal_7 --configure-only --legacy-root=http://drupal7.example.com
+```
+This command configures (but does not run) the migrations from Drupal 6 or 7 to Drupal 8. In this example, the Drupal 8 site is named `my-drupal-8-site` and the command is running on the `dev` environment. The `--legacy-db-key` parameter indicates how to get the login credentials to the source Drupal 6 or 7 database. In our example, we use the [Terminus secrets](https://github.com/pantheon-systems/terminus-secrets-plugin) plugin to supply the connection info. [See our blog post for more information on how this flag is used](https://pantheon.io/blog/running-drupal-8-data-migrations-pantheon-through-drush). The `--legacy-root` flag lets Drupal 8 know from where it can grab images and other uploaded media assets.
+
+The following command generates a report on how many items have been imported by each migration:
+```
+terminus drush my-drupal-8-site.dev -- migrate-status
+```
+
+The following command runs the migration configured via `drush migrate-upgrade`:
```

Comment:
Yes, `drush migrate-import` is what runs the migration. `drush migrate-upgrade` can both configure and run migrations. Above I include the `--configure-only` flag. I could do that here too so this line would read as 

> The following command runs the migration configured via `drush migrate-upgrade --configure-only`: